frontend/src/components/LineChart/LineChart.js

```diff
@@ -1,38 +1,69 @@
 import { PropTypes } from 'prop-types'
-import { useMemo, useRef } from 'react'
+import { useCallback, useMemo, useRef } from 'react'
 import { useTheme } from '@mui/system'
-import { VictoryAxis, VictoryChart, VictoryLine, VictoryScatter, VictoryTooltip } from 'victory'
+import {
+  VictoryAxis,
+  VictoryChart,
+  VictoryLine,
+  VictoryScatter,
+  VictoryTooltip,
+  VictoryVoronoiContainer,
+} from 'victory'
 import Box from '@mui/material/Box'
+import { useIntl } from 'react-intl'
 
-import { nFormatter } from 'utils/helpers'
-import { getLineChartDomain, useVictoryTheme, useClientSize } from 'utils/victory'
+import { formatTickNumber, getLineChartDomain, useVictoryTheme, useClientSize } from 'utils/victory'
+import { ChartAxisType } from 'utils/propTypes'
 import ChartFlyOut from 'components/ChartFlyOut'
 
 const LINE_CHART_RATIO = 0.75
 
-const LineChart = ({ data, title, xAxisLabel, yAxisLabel }) => {
+const LineChart = ({ data, title, xAxis, yAxis }) => {
+  const { formatNumber } = useIntl()
   const theme = useTheme()
   const domain = useMemo(() => getLineChartDomain(data, LINE_CHART_RATIO), [data])
   const victoryTheme = useVictoryTheme(theme)
-
   const ref = useRef()
   const size = useClientSize(ref)
 
+  const handleXTickFormat = useCallback(
+    t => {
+      return formatTickNumber(t, formatNumber, xAxis)
+    },
+    [formatNumber, xAxis],
+  )
+
+  const handleYTickFormat = useCallback(
+    t => {
+      return formatTickNumber(t, formatNumber, yAxis)
+    },
+    [formatNumber, yAxis],
+  )
+
   return (
-    <Box ref={ref} sx={{ columns: 1, marginTop: '-50px' }}>
-      <VictoryChart {...size} theme={victoryTheme}>
-        <VictoryAxis />
-        <VictoryAxis dependentAxis domain={domain} tickFormat={t => nFormatter(t)} />
+    <Box ref={ref} sx={{ columns: 1, '& svg': { overflow: 'visible', position: 'relative', zIndex: 1 } }}>
+      <VictoryChart
+        {...size}
+        theme={victoryTheme}
+        domainPadding={{
+          x: [0, 15],
+          y: [0, 20],
+        }}
+        containerComponent={
+          <VictoryVoronoiContainer
+            labels={() => ' '}
+            labelComponent={
+              <VictoryTooltip
+                constrainToVisibleArea
```

Comment:
It looks like the tooltip clipping issue I mentioned in my last review only happens in Safari and is related to the tooltip's DOM element sitting inside the CSS `columns` hierarchy. Safari appears to use an optimization for `columns` where it renders one long column texture and then wraps the result. If I switch to a 3 column layout, I can even make the [absolutely positioned tooltip wrap across columns](https://user-images.githubusercontent.com/244952/156217795-1e707c9f-7274-43d1-ad45-5865d2a46cb7.png).

`constrainToVisibleArea` seems to have no effect on the tooltip's positioning or clipping. Maybe it only works on tooltips without custom components?

The only solutions I can think of are:
1. Find another CSS method for wrapping content across 2 columns besides the `columns` property. That might be `grid`, but in my experience `grid` needs a predefined column height to know when to wrap.
2. Use a portal to opt the tooltip out of the `columns` hierarchy, making the tooltip's DOM element a child of the body element and calculate its position manually. There are a couple of examples of this in Victory's GitHub issues: [example one](https://github.com/FormidableLabs/victory/issues/576#issuecomment-342299341) and [example two](https://github.com/FormidableLabs/victory/issues/576#issuecomment-392499716).